index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Document</title>
```

Comment:
This is a good catch. Well done on pointing this out ✨. Try to be specific, concise, and helpful in your review comments. This will make it easier for the person to understand and act on your suggestions. 
- Start by thanking the person for their work and for considering your suggestion. This helps to establish a positive and constructive tone for the rest of the comment.

- Provide some context for why you are suggesting that the person change the default title. For example, you could mention that a strong title can help to make the project look more polished and professional, or that it can impact the website's search engine optimization (SEO).
- Offer some specific tips or guidelines for choosing a good title. This can help the person to understand what you are suggesting and how to implement it.
- Consider including a question or two to encourage the person to ask for more help or clarification if needed. This can help to make the review process more collaborative and interactive.

I hope that helps, let me know if you have any questions 🙂